testplan/exporters/testing/pdf/renderers/entries/assertions.py

```diff
@@ -1,5 +1,9 @@
 import re
-from cgi import escape
+
+try:
+    from html import escape
+except ImportError:
+    from cgi import escape
```

Comment:
What about this instead:

```py
try:
    from html import escape as _orig_escape
    from functools import partial
    escape = partial(_orig_escape, quote=False)
except ImportError:
    from cgi import escape
```

Reason being that we don't have any tests for how the PDF looks, and escaping quotes could have unforeseen consequences for how the PDF ends up looking.